sample/android/src/main/java/co/nimblehq/kmm/template/di/modules/AppModule.kt

```diff
@@ -0,0 +1,12 @@
+package co.nimblehq.kmm.template.di.modules
+
+import co.nimblehq.kmm.template.util.DispatchersProvider
+import co.nimblehq.kmm.template.util.DispatchersProviderImpl
+import org.koin.core.module.dsl.singleOf
+import org.koin.dsl.bind
+import org.koin.dsl.module
+
+val appModule = module {
+    singleOf(::DispatchersProviderImpl) bind DispatchersProvider::class
+
```

Comment:
Redundant line

Reply:
@kaungkhantsoe I tried to [clean up unnecessary stuff from the default AppModule](https://github.com/nimblehq/kmm-templates/pull/32/files/d745c42098768783515499ec8efc9b529f12794d#diff-a73eeeff37b5a3054b0c150ce7167b5c4191336a80bd149557b979eba8f23f3cR49-R54). However, it's not easy to remove this redundant line by Bash script. I will remove it when working on building the main Kscript by this task https://github.com/nimblehq/kmm-templates/issues/30 🙏